### ISSUE TITLE
Design and Name utilities 

### ISSUE THREAD

**Issue by amyers1997** [baclevenger/spacegame#9]

**Comment by jakefist** [comment]
Could you explain?

**Comment by amyers1997** [comment]
We need to decide what buildings we are going to have in the game, we need a list with their names, functions, and designs

**Comment by jakefist** [comment]
Agreed. I think we will call them installations. We will also need to link this to research trees if you build them. Can you trade installations with other players?
